plugins/single_plugins/oswitch.cpp

```diff
@@ -24,6 +24,9 @@ class wayfire_output_manager : public wf::plugin_interface_t
 
             switch_output = [=] (wf_activator_source, uint32_t)
             {
+                if (!output->activate_plugin(grab_interface))
```

Comment:
Not sure that I like this check, but I am also not sure how we can implement it (maybe we'll have to extend the `activate_plugin` function). Use-case I have in mind: we can have Expo open on the focused output, and it is perfectly fine to use a keybinding to switch to the other output.